exercises/practice/pov/pov.go

```diff
@@ -1,23 +1,52 @@
 package pov
 
-// Define the Graph type here.
+// Define the Tree type here
 
-func New() *Graph {
-	panic("Please implement the New function")
+type Tree struct{}
```

Comment:
Line 3 and 5 are a bit contradictory. In other exercises we only have the equivalent of line 3 and nothing like line 5 so the student needs to decide that they want a struct here. Alternatively, can a comment like "Add the needed fields here" as body of the struct instead of line 3.

Reply:
Thanks. I like the second option.